src/scenic/simulators/carla/simulator.py

```diff
@@ -161,6 +161,11 @@ def createObjectInSimulator(self, obj):
 		rot = utils.scenicToCarlaRotation(obj.heading)
 		transform = carla.Transform(loc, rot)
 
+		# Color
+		c = obj.color
+		c_str = f'{int(c.r*255)},{int(c.g*255)},{int(c.b*255)}'
+		blueprint.set_attribute('color', c_str)
```

Comment:
Does this change the color of all vehicles using the same blueprint?

Reply:
It should only affect the actor that is spawned with this blueprint in the next statement (line 170), so there shouldn't be a problem.

Reply:
OK, thanks.